package.json

```diff
@@ -24,56 +24,57 @@
   ],
   "scripts": {
     "build": "webpack --progress --config resources/assets/build/webpack.config.js",
-    "build:production": "webpack --env.production --progress --config resources/assets/build/webpack.config.js",
+    "build:production": "webpack --mode=production --progress --config resources/assets/build/webpack.config.js",
     "build:profile": "webpack --progress --profile --json --config resources/assets/build/webpack.config.js",
-    "start": "webpack --hide-modules --watch --config resources/assets/build/webpack.config.js",
+    "start": "webpack --watch --config resources/assets/build/webpack.config.js",
     "rmdist": "rimraf dist",
     "lint": "npm run -s lint:scripts && npm run -s lint:styles",
     "lint:scripts": "eslint resources/assets/scripts resources/assets/build",
     "lint:styles": "stylelint \"resources/assets/styles/**/*.{css,sass,scss,sss,less}\"",
     "test": "npm run -s lint"
   },
   "engines": {
-    "node": ">= 8.0.0"
+    "node": ">= 12.18"
   },
   "devDependencies": {
+    "@soda/friendly-errors-webpack-plugin": "^1.8.0",
     "autoprefixer": "^10.0.1",
     "browser-sync": "^2.26.13",
-    "browsersync-webpack-plugin": "^0.6.0",
+    "browser-sync-webpack-plugin": "^2.3.0",
     "bs-html-injector": "~3.0",
-    "buble-loader": "^0.4.1",
-    "cache-loader": "~1.2.5",
-    "clean-webpack-plugin": "^0.1.18",
-    "copy-globs-webpack-plugin": "^0.2.0",
-    "css-loader": "^0.28.11",
-    "cssnano": "^4.0.5",
-    "eslint": "~4.19.1",
-    "eslint-loader": "^4.0.2",
+    "buble": "^0.20.0",
+    "buble-loader": "^0.5.1",
+    "clean-webpack-plugin": "^3.0.0",
+    "copy-webpack-plugin": "^9.0.0",
+    "css-loader": "^5.0.1",
+    "cssnano": "^5.0.5",
+    "eslint": "^7.16.0",
     "eslint-plugin-import": "^2.14.0",
-    "extract-text-webpack-plugin": "~3.0.2",
+    "eslint-webpack-plugin": "^2.4.1",
     "file-loader": "^6.2.0",
-    "friendly-errors-webpack-plugin": "^1.6.1",
     "imagemin-mozjpeg": "^9.0.0",
     "imagemin-webpack-plugin": "^2.4.2",
     "import-glob": "~1.5",
-    "node-sass": "^5.0.0",
-    "postcss-loader": "^4.0.4",
-    "postcss-safe-parser": "^5.0.2",
-    "resolve-url-loader": "^3.1.2",
+    "mini-css-extract-plugin": "^1.3.3",
+    "postcss": "^8.2.2",
+    "postcss-loader": "^6.1.0",
+    "postcss-safe-parser": "^6.0.0",
+    "resolve-url-loader": "^4.0.0",
     "rimraf": "^3.0.2",
-    "sass-loader": "~6.0",
-    "style-loader": "^0.23.1",
+    "sass": "^1.32.0",
```

Comment:
Ran into an issue with sass v1.40.0, when using Bootstrap 4 [reported in discourse](https://discourse.roots.io/t/sage-9-1-please-test/20067/100?u=dalepgrant)

Restricting the version range to `"sass": ">=1.32.0 <=1.39.2",` fixes the issue.

Reply:
@dalepgrant: Thanks for the feedback. I explicitly bumped `sass` to `1.41.0` to ensure this issue won't occur when updating.